src/components/_global/BalAsset/BalAsset.vue

```diff
@@ -1,96 +1,100 @@
 <template>
-  <img
-    v-if="iconSRC && !error"
-    :src="iconSRC"
+  <component
```

Comment:
Should move the template beneath the Javascript when moving from defineComponent to setup.